frontend/src/pages/VacationPlanner.tsx

```diff
@@ -343,9 +380,43 @@ export const VacationPlanner = () => {
             </button>
           </div>
         </div>
-        <h3>Time entries</h3>
-        <div>{JSON.stringify(Object.values(redmineGroups))}</div>
-        <div>{JSON.stringify(vacationEntries)}</div>
+        <div className="group-select-wrapper">
+          <select
+            className="col-3 footer-field"
+            name="activity"
+            id="select-activity"
+            onChange={(e) => {
+              const groupId = e.target.value;
+              const group = redmineGroups.find((group) => {
+                return group.id == groupId;
+              });
+              setSelectedGroup(group);
+            }}
+            style={{ width: "50%" }}
```

Comment:
This is copied from somewhere else in our code, right? I've wanted to clean it up for such a long time and now it gets copied... >< I think we should try to avoid inline styles if there isn't a really good reason to use it which I don't see here.